po/common/common.pot

```diff
@@ -5428,3 +5428,207 @@ msgstr "Type of the product photo"
 msgctxt "image_other_type_description"
 msgid "If you use the same column on multiple lines to provide images URLs for a single product, you can use this field to indicate the type of the image: front, ingredients, nutrition or packaging."
 msgstr "If you use the same column on multiple lines to provide images URLs for a single product, you can use this field to indicate the type of the image: front, ingredients, nutrition or packaging."
+
+msgctxt "ecoscore_missing_data_warning"
+msgid "<strong>Warning: some information necessary to calculate the Eco-Score with precision is not provided (see the details of the calculation below)</strong><br><br>You can edit the product to add the missing information.<br><br>If you are the manufacturer of this product, you can send us the information with our <a href=\"https://world.pro.openfoodfacts.org\">free platform for producers</a>."
```

Comment:
The <strong> <br> etc. tags should stay in the template.

Reply:
fixed